tests/foreman/cli/test_contentview.py

```diff
@@ -3962,3 +3962,23 @@ def test_positive_inc_update_should_not_fail(self, module_org):
         assert result[2]
         content_view = ContentView.info({'id': content_view['id']})
         assert '1.1' in [cvv_['version'] for cvv_ in content_view['versions']]
+
+    @pytest.mark.tier1
+    def test_positive_pulp_workers_running_state(self, target_sat):
+        """Ensure that 6 pulpcore-workers are in active (running) state
+
+        :BZ: 1848111
+
+        :id: f0b8b354-1a30-41c3-93d7-fb51e99c82b6
+
+        :customerscenario: true
+
+        :expectedresults: 6 pulpcore-workers are in active (running) state
+
+        :SubComponent: Pulp
+        """
+        output = target_sat.execute('systemctl status pulpcore-worker* | grep Active')
```

Comment:
Would suggest using f-m CLI to check service status with `brief` and `only` options rather than systemctl, where you could validate the RC of the command instead of splitting stdout
```suggestion
        output = target_sat.cli.Service.status({'brief': True, 'only': 'pulpcore-worker*'})
```

Reply:
This bleeds into 'testing' the CLI for services - not a bad idea, but may influence how the tests are organized. I've suggested verifying the pulpcore-worker states in the existing pulp_filesystem module, if using `Service.status` it would start to make more sense that the test is in a module for services CLI, and pulpcore-worker is just one of the services being validated.